mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/UtbetalingService.kt

```diff
@@ -361,6 +362,73 @@ class UtbetalingService(
         delutbetaling
     }
 
+    fun oppdaterFakturaStatus(
+        fakturanummer: String,
+        nyStatus: FakturaStatusType,
+        fakturaStatusSistOppdatert: LocalDateTime?,
+    ) = db.transaction {
+        val originalDelutbetaling = queries.delutbetaling.getOrError(fakturanummer)
+        if (originalDelutbetaling.fakturaStatusSistOppdatert != null &&
+            fakturaStatusSistOppdatert != null &&
+            originalDelutbetaling.fakturaStatusSistOppdatert.isAfter(
+                fakturaStatusSistOppdatert,
+            )
+        ) {
+            return
+        }
```

Comment:
Er ikke denne sjekken litt unødvendig? Vil vel aldri skje normalt? Er det for å slippe å oppdatare hvis man skulle relaste topic?